pkg/subctl/resource/schedule_pod.go

```diff
@@ -106,6 +106,15 @@ func (np *NetworkPod) schedulePod() error {
 		},
 	}
 
+	if np.Config.Networking == framework.HostNetworking {
+		networkPod.Spec.Containers[0].SecurityContext = &v1.SecurityContext{
+			Capabilities: &v1.Capabilities{
+				Add:  []v1.Capability{"CAP_NET_ADMIN", "CAP_NET_RAW"},
```

Comment:
Capabilities are supposed to be defined without the leading `CAP_`: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

Reply:
Thanks a lot @skitt. Interestingly, it was working fine on OCP even with leading `CAP_`